internal/serve/httphandler/profile_handler_test.go

```diff
@@ -454,6 +459,162 @@ func Test_ProfileHandler_PatchOrganizationProfile(t *testing.T) {
 		assert.Equal(t, "-03:00", org.TimezoneUTCOffset)
 		assert.Equal(t, imgBuf.Bytes(), org.Logo)
 	})
+
+	t.Run("updates organization's SMS Registration Message Template", func(t *testing.T) {
```

Comment:
Sorry. I didn't understand. Could you elaborate?

Reply:
I mean that when I read a chunk of code like the following, it's hard to follow because there are so many blank lines. A lot of these calls that are separated by blank lines are basically part of the same thing, so they could be organized more closely together.

Does that make sense?

https://github.com/stellar/stellar-disbursement-platform-backend/blob/a45313d2d69d5a18dcbfd7366ff87a706fdedf9c/internal/serve/httphandler/profile_handler_test.go#L477-L534

Reply:
Sure. I'll group related things. Thank you.